app/client/contract.ts

```diff
@@ -535,10 +535,26 @@ export const definition = {
         createdAt: z.date(),
         debug: z.boolean(),
         enableCustomAuth: z.boolean(),
-        handleCustomAuthFunction: z.string(),
+        handleCustomAuthFunction: z.string().nullable(),
         enableKnowledgebase: z.boolean(),
         lastPingAt: z.date().nullable(),
         isDemo: z.boolean(),
+        machines: z.array(
+          z.object({
+            id: z.string(),
+            lastPingAt: z.date().nullable(),
+            ip: z.string().nullable(),
+            sdkVersion: z.string().nullable(),
+            sdkLanguage: z.string().nullable(),
+          })
+        ),
+        services: z.array(
```

Comment:
Should we make this the same structure as `listServices` for consistency

Reply:
I was thinking about it, but I think this is more akin to the timeline endpoint where we show some minimal data and the "liveness". I think `listServices` would grow to have more data inside.